common/src/main/java/com/wynntils/functions/EnvironmentFunctions.java

```diff
@@ -20,6 +20,12 @@ public static class CappedMemFunction extends Function<CappedValue> {
         public CappedValue getValue(FunctionArguments arguments) {
             return new CappedValue(SystemUtils.getMemUsed(), SystemUtils.getMemMax());
         }
+
+        @Override
+        public List<String> getAliases() {
+            // FIXME: These aliases are a bit backwards, let's clean it up in the future
```

Comment:
I plan to fix this in an upcoming PR, fixing all these weird names. I think this is fine to leave.

Reply:
I used to think we should keep parity with the old names, but they kind of suck all of them, so I'm more leaning towards giving stuff new, good, names. We're changing the syntax away from %% anyway, so there is not much to convert over.

For functions returning capped values, I don't think we should have any "uncapped" variants. So there should just be a "mana", etc.

Reply:
I agree. We need to fix it in a singular, breaking PR